benchmark/UnsortedAndSortedJoinBenchmark.cpp

```diff
@@ -0,0 +1,360 @@
+// Copyright 2015, University of Freiburg,
```

Comment:
This doesn't really have anything to do with the functionality of the branch. Delete it, before you finish.

Reply:
Got rid of the cmake entry in [34779da](https://github.com/ad-freiburg/qlever/pull/860/commits/34779dad2b792e0cde0be871bc0d007b847123bf), but kept the file for now, until I can move it to it's own branch later.